### PR TITLE
fix spelling in case of no items

### LINKED ISSUE #756: Spelling error in blocks empty-state text
## Description

Empty state should say "There are no items"

Screen Shot 2020-02-14 at 9 36 11 AM

### Steps to reproduce

remove all blocks

### Expected result

"Are"... not "is" or " ain't" :)

### PR DESCRIPTION
**Pull request by zivkaziv**
Fix #756 
Changed spelling from  "There's no items" to  "There are no items

### CODE REVIEW

**Comment by zivkaziv** [comment]
I'm not sure why Build and publish Auto generated API documentation failed..
Can someone help me with it?

File: packages/demo-gatsby/data/author.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "firstName": "Nolan",
-  "lastName": "Phillips",
+  "firstName": "Ziv",
+  "lastName": "Kalderon",
```
Comment (DirtyF) [inline packages/demo-gatsby/data/author.json]: Not relevant to this PR :)

**Comment by DirtyF** [comment]
@zivkaziv I guess, this action doesn't run on forks.

**Comment by zivkaziv** [comment]
Do you want me to recreate this PR?

File: packages/demo-gatsby/data/author.json (hunk 1 above)
Comment (zivkaziv) [reply packages/demo-gatsby/data/author.json]: Pushed fix for it.. Sorry.. I'm not sure why I did push on it

**Comment by ncphillips** [comment]
I’m pretty confident this will pass the CI 😉

**Pull request merged**